src/cli.js

```diff
@@ -176,3 +177,8 @@ switch (args._[0]) {
     console.error(`Error: Invalid command "${args._[0]}"\n${usage}`);
     process.exit(1);
 }
+
+// remove me when node.js makes this the default behavior
+process.on("unhandledRejection", e => {
+  throw e;
+});
```

Comment:
Where is this coming from? We should always try to handle the promises we create. Agreed that it's a good thing to add as a safeguard

Reply:
Good point - I've handled the rejection rather.

Reply:
I do like the safeguard. I strongly dislike Node's default mechanism

Reply:
Sure added it back too!